packages/http-server/src/server.ts

```diff
@@ -109,6 +109,10 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
           E.tryCatch(() => {
             if (output.headers) reply.headers(output.headers);
 
+            // Since we are buffering the whole response body, do not relay the "transfer-encoding" header if it exist.
+            // It will be combined with the "Content-Length" header which is not valid and will cause certain clients to fail(etc. Postman)
+            reply.removeHeader('transfer-encoding');
```

Comment:
This should only be happening if the `proxy` option is set

Reply:
Did this by stripping the headers in the forwarder instead